myScript.js

```diff
@@ -24,3 +24,704 @@ function activeMenu() {
+        </div>
+      </div>
+      </div>
+  `;
+
+  document.getElementById('overlay').style.display = 'block';
+  document.getElementById('popup').style.display = 'block';
+  const buttonClose = document.getElementById('popupclose');
+  function closePopup() {
+    document.getElementById('overlay').style.display = 'none';
+    document.getElementById('popup').style.display = 'none';
+  }
+
+  buttonClose.addEventListener('click', closePopup);
+}
+
+project5.addEventListener('click', project5content);
+
+function project6content() {
+  mobilePopup.innerHTML = `
+  <div id="overlay" ></div>
+      <div id="popup">
+        <div class="popupcontrols">
+          <i class="fa-solid fa-x" id="popupclose"></i>
+        </div>
+        <div class="popupcontent">
+          <h1>${arr[5].name}</h1>
+        
+        <div class="button-area">
+          <button class="popuptopbutton">${arr[5].technologies[0]}</button>
+          <button class="popuptopbutton">${arr[5].technologies[1]}</button>
+          <button class="popuptopbutton">${arr[5].technologies[2]}</button>
+        </div>
+
+        <div class="projectpreview">
+          <img src=${arr[5].image[0]} alt="project1" class="mainpopupimage"/>
+          <div class="mobile-popup">
+            <img src=${arr[5].image[0]} alt="project1" class="subpopupimages" />
+            <img src=${arr[5].image[1]} alt="project1" class="subpopupimages" />
+            <img src=${arr[5].image[2]} alt="project1" class="subpopupimages" />
+            <img src=${arr[5].image[3]} alt="project1" class="subpopupimages" />
+          </div>
+        </div>
+        <p class="paragraphpopup">
+        ${arr[5].description}
+        </p>
+        <div class="popupbottombtn">
+          <button href=${arr[5].livelink}>
+            See Live <i class="fa-solid fa-arrow-right-to-bracket"></i>
+          </button>
+        </div>
+        <div class="popupbottombtn">
+          <button href=${arr[5].sourcelink}>
+            See Source <i class="fa-solid fa-arrow-right-to-bracket"></i>
+          </button>
+        </div>
+      </div>
+      </div>
+  `;
+
+  document.getElementById('overlay').style.display = 'block';
+  document.getElementById('popup').style.display = 'block';
+  const buttonClose = document.getElementById('popupclose');
+  function closePopup() {
+    document.getElementById('overlay').style.display = 'none';
+    document.getElementById('popup').style.display = 'none';
+  }
+
+  buttonClose.addEventListener('click', closePopup);
+}
+
+project6.addEventListener('click', project6content);
+
+const firstPopup = document.getElementById('project-popup');
+const secondPopup = document.getElementById('laptopimage');
+const thirdPopup = document.getElementById('project-popup2');
+const fourthPopup = document.getElementById('projectimg1');
+const fifthPopup = document.getElementById('projectimg4');
+
+const desktopPopup = document.getElementById('popupdesktopd');
+
+function project1dcontent() {
```

Comment:
- [ ] Make this to be your main pop-up window, render data dynamically and delete the other below.